### ISSUE TITLE
Documentation for tcponudp

### ISSUE THREAD

**Issue by louisabraham** [RetroShare/RetroShare#1579]
Hi, if tcponudp does what I think it does (execute the TCP protocol through a UDP socket), I'm greatly interested in using it.
However, I didn't find any documentation, instructions or simple example.
Do you think you could produce some? Or do you know alternatives?

**Comment by G10h4ck** [comment]
Yeah AFAIK tcponudp is an user space tcp-like transport protocol implemented on top of UDP, it basically implements a BIO struct so OpenSSL can operate over an UDP socket like it was a TCP one. I have fixed some bug in it some time ago, but I don't have a completely clear understanding of it. Any effort on documenting it is welcome.
